suites/pacific/cephfs/tier_1_cephfs_mirror.yaml

```diff
@@ -0,0 +1,141 @@
+                      label: mon
+              - config:
+                  command: apply
+                  service: osd
+                  args:
+                    all-available-devices: true
+              - config:
+                  command: shell
+                  args: # arguments to ceph orch
+                    - "ceph fs volume create cephfs"
+              - config:
+                  command: apply
+                  service: mds
+                  base_cmd_args: # arguments to ceph orch
+                    verbose: true
+                  pos_args:
+                    - cephfs                        # name of the filesystem
+                  args:
+                    placement:
+                      nodes:
+                        - node4
+                        - node5
+              - config:
+                  command: apply
+                  service: cephfs-mirror
+                  args:
+                    placement:
+                      nodes:
+                        - node6
+        ceph2:
+          config:
+            verify_cluster_health: true
+            steps:
+              - config:
+                  command: bootstrap
+                  service: cephadm
+                  args:
+                    registry-url: registry.redhat.io
+                    mon-ip: node1
+                    orphan-initial-daemons: true
+                    skip-monitoring-stack: true
+              - config:
+                  command: add_hosts
+                  service: host
+                  args:
+                    attach_ip_address: true
+                    labels: apply-all-labels
+              - config:
+                  command: apply
+                  service: mgr
+                  args:
+                    placement:
+                      label: mgr
+              - config:
+                  command: apply
+                  service: mon
+                  args:
+                    placement:
+                      label: mon
+              - config:
+                  command: apply
+                  service: osd
+                  args:
+                    all-available-devices: true
+              - config:
+                  command: shell
+                  args:                             # arguments to ceph orch
+                    - "ceph fs volume create cephfs"
+              - config:
+                  command: apply
+                  service: mds
+                  base_cmd_args:                    # arguments to ceph orch
+                    verbose: true
+                  pos_args:
+                    - cephfs                        # name of the filesystem
+                  args:
+                    placement:
+                      nodes:
+                        - node4
+                        - node5
+      desc:  CephFS Mirror cluster deployment using cephadm
+      destroy-clster: false
+      module: test_cephadm.py
+      name: deploy cephfs-mirror
```

Comment:
Could we add a polarion ID? This is a new deployment... helps keep track with test cases